versions.gradle

```diff
@@ -7,10 +7,10 @@
 
 // casual-java group and version
 group = 'se.laz.casual'
-version = '1.0.82-beta'
+version = '1.0.0'
```

Comment:
Accidental reset?

Reply:
This change is intentional. Since I'm making breaking api changes to CasualQueueApi (which propagates to CasualCallerApi) the version should reasonably change on the major. After a previous discussion with @AdjacentDoor we concluded that it could make sense to use the occasion instead to drop the beta and just call this 1.0.0, while casual-caller was bumped to 2.0.0 since it already had a "final" version 1.*